apps/api/models.py

```diff
@@ -102,24 +105,50 @@ class APIKey(models.Model):
     class Meta:
         db_table = 'api_key'
 
+    def __unicode__(self):
+        return (
+            u'<{cls} user={user}, type={type}, key={key} secret=...>'
+            .format(cls=self.__class__.__name__, key=self.key,
+                    type=self.type, user=self.user))
+
     @classmethod
     def get_jwt_key(cls, **query):
```

Comment:
actually, the auth view needs to do a lookup by issuer key (from the JWT) because we don't know the user at that point. I think rather than make two separate methods I'll just leave it as is. The main purpose of this one was to always return JWT-type keys and active ones by default.